apps/charterafrica/src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -86,7 +86,10 @@ const OpportunityCard = React.forwardRef(function OpportunityCard(props, ref) {
             variant="p1"
             elements={excerpt}
             sx={(theme) => ({
-              height: `calc(${theme.typography.p1.fontSize}px * 3)`,
+              maxHeight: `calc(${
+                theme.typography.p1.fontSize * theme.typography.p1.lineHeight
+              }px * 3)`,
+              overflow: "hidden",
```

Comment:
1. lineClamp should handle overflow,
2. switch to maxHeight means if element has 1 line, it will take height 1 line, if 2, it will take height of 2, etc. Doesn't this mess up the alignment? We want all card to align perfectly.